tests/rptest/tests/memory_stress_test.py

```diff
@@ -36,22 +36,28 @@ def setUp(self):
         # enabling each test case to customize its ResourceSettings
         pass
 
-    @ok_to_fail  # until the fix is delivered
     @cluster(num_nodes=5)
     @skip_debug_mode
-    def test_fetch_with_many_partitions(self):
+    @parametrize(memory_share_for_fetch=0.05)
+    @parametrize(memory_share_for_fetch=0.3)
+    @parametrize(memory_share_for_fetch=0.95)
```

Comment:
nice.

i sort of expected to see the ducktape tests expanded to cover a wider parameter range of configurations, fetch request configurations, and data sizes present in the underlying topic. wdyt? maybe in follow up? my concern is that are covering enough of the edge cases in the new code and its math.

Reply:
The other parameters are selected so that the test creashed redpanda before the change, that's why I left them intact. I think that the coverage is good enough - the unit test deliberately covers all cases in math. However it may make sense to have more cases cover e.g. huge and tiny batch sizes, and how they would work with different default batch size setting.